nautobot/core/views/mixins.py

```diff
@@ -193,6 +193,9 @@ def get_return_url(self, request, obj=None):
             # Model has no get_absolute_url() method or no reverse match
             pass
 
+        if default_return_url is not None:
```

Comment:
Is there a case where `default_return_url` is not None?

Reply:
https://github.com/nautobot/nautobot/pull/5054/files#diff-caf8a412a9f30295cacfae888c0fbc1bdc2a7cd2643df55f444414f363113a7bR790 - I didn't want to change the default return url at the view level here, just for this one specific case.